src/middlewared/middlewared/plugins/support.py

```diff
@@ -223,7 +223,7 @@ async def new_ticket(self, job, data):
             if i not in data:
                 raise CallError(f'{i} is required', errno.EINVAL)
 
-        data['version'] = (await self.middleware.call('system.version')).split('-', 1)[-1]
+        data['version'] = f'SCALE-{await self.middleware.call("system.version", False)}'
```

Comment:
Is that `PRODUCT` constant is really necessary? If yes, why don't we use it here?

Reply:
Just missed in it the original PR. I'll update to use it here.